src/pug/index.pug

```diff
@@ -229,6 +229,17 @@ html(lang='en')
                             a.btn.btn-dark.btn-social.mx-2(href='https://api.pdok.nl/lv/bgt/ogc/v1/api?lang=en-US' title='OpenAPI definition' target="_blank")
                                 i.fa.fa-book-open 
 
+                    .col-lg-4
+                        .team-member
+                            img.mx-auto.rounded-circle(src='assets/img/ca-eccc-msc.png', alt='...')
+                            h4 MSC GeoMet - GeoMet-OGC-API
+                            p.text-muted Powered by pygeoapi
+                            p.text-muted GeoMet-OGC-API provides public access to the Meteorological Service of Canada (MSC) and Environment and Climate Change Canada (ECCC) data via interoperable web services and application programming interfaces (API). Through open standards, users can freely and quickly access thousands of real-time and archived weather, climate and water datasets and products and integrate them in their domain-specific applications and decision support systems. Users can build mobile apps, create interactive web maps, and display and animate MSC data in desktop software. MSC GeoMet also enables on demand raw data clipping and reprojection, on demand format conversion and custom visualization.
```

Comment:
@tomkralidis Thank You! Would it be possible to make the description a bit smaller, just to be more aligned with other ones? 🙏🏽 I think that less than 390 characters would be ideal

Reply:
Thanks, fixed